src/pages/ViewEmailPage/index.jsx

```diff
@@ -22,10 +24,21 @@ function ViewEmailPage() {
                 setSubject(emailData.data.email.subject)
                 setBody(emailData.data.email.body)
                 setDate(emailData.data.email.date_created)     
-                setIsDeleted(emailData.data.email.deleted)           
+                setIsDeleted(emailData.data.email.deleted)
+                setIsRead(emailData.data.email.read)           
             })
     }
 
+    function markEmailRead() {
+        fetch(`https://email-client-api.dev.io-academy.uk/emails/${id}`, {
```

Comment:
It would be good to only send this request if the email is not read - since you already have isRead it should be a case of just putting this fetch in an if statement. 

It's not a big deal since the API doesn't really do anything if the email is already read, it's just a nice thing to do from an efficiency perspective.